cider/exec/nextgen/jitlib/llvmjit/LLVMJITFunction.cpp

```diff
@@ -119,7 +126,8 @@ llvm::Value* createConstantImpl(llvm::LLVMContext& context, std::any value) {
   }
 }
 
-JITValuePointer LLVMJITFunction::createConstant(JITTypeTag type_tag, std::any value) {
+JITValuePointer LLVMJITFunction::createLiteralImpl(JITTypeTag type_tag,
+                                                   const std::any& value) {
   llvm::Value* llvm_value = nullptr;
   switch (type_tag) {
```

Comment:
can we just `llvm_value = createConstantImpl<decltype(type_tag)>(getLLVMContext(), value);`

Reply:
It's unfeasible, ```decltype(type_tag)``` will be deducted as ```enum cider::jitlib::JITTypeTag```.